portfolio/src/main/webapp/css/aboutme.css

```diff
@@ -7,59 +7,60 @@ body {
     margin: 0;
 }
 
-#container {
+
+.container {
     align-items: center;
     display: flex;
     flex-direction: column;
     font-family: Garamond;
-    height: 100%;
-    width: 100%;
+    font-size: 1.25em;
 }
 
-#wide-top {
+.top-image {
     background: url(../images/me.jpg) bottom no-repeat;
     background-size: cover;
     flex-shrink: 0;
     height: 70%;
     width: 100%;
 }
 
-#text-content {
-    font-size: 14pt;
-    height: auto;
-    padding-bottom: 1em;
-    width: 30%;
+.text-content {
+    padding: 1.5em;
+    padding-left: 3em;
+    padding-right: 3em;
+    /* background: red; */
 }
 
-#text-content h1{
+.title-font {
     font-family: Verdana, Tahoma, sans-serif;
     font-weight: bold;
-    padding-top: 5%;
 }
 
-#footer {
-    font-family: Verdana, Tahoma, sans-serif;
-    height: auto;
-    padding-bottom: 2em;
+.footer {
+    padding-bottom: 1.5em;
     text-align: center;
-    width: auto;
+    width: 100%;
 }
 
-#add-comment {
-  text-align: right;
-}
-
-#comments {
-    padding-top: 1em;
-    text-align: left;
-}
 
 .custom-btn {
     border-radius: 0;
     border: 2px solid black;
     color: black;
 }
 
+#comments {
+  text-align: left;
+  font-weight: bold;
```

Comment:
nit:Order the properties alphabetically/.

Reply:
Done: 979f7f6